src/StatusQ/Controls/StatusDropdown.qml

```diff
@@ -0,0 +1,49 @@
+import QtQuick 2.13
+import QtGraphicalEffects 1.13
+import QtQuick.Controls 2.14 as QC
+
+import StatusQ.Core.Theme 0.1
+
+/*!
+   \qmltype StatusDropdown
+   \inherits Popup
+   \inqmlmodule StatusQ.Popups
+   \since StatusQ.Popups 0.1
+   \brief The StatusDropdown provides a template for creating dropdowns.
+
+   NOTE: Each consumer needs to set the x and y postion of the dropdown.
+
+   Example of how to use it:
+
+   \qml
+        StatusDropdown {
+            x: root.x + margins
+            y: root.y + margins
+            contentItem: ColumnLayout {
+                ...
+            }
+        }
+   \endqml
+
+   For a list of components available see StatusQ.
+*/
+QC.Popup {
```

Comment:
You already have a dedicated QC.MenuItem component, I suggest creating this new custom one on something pre-existing (MenuItem already inherits from Popup ;)

Reply:
I'm not sure about using `QC.MenuItem` as a base class since this component is just a `popup` container with a specific background, a dropdown container. If `MenuItem` inherits from `Popup` but has more functionality, we will be inheriting more functionality than needed for this specific case. WDYT?

Reply:
Sorry, of course I meant a `Menu`, that one is the container, see here: https://doc.qt.io/qt-5/qml-qtquick-controls2-menu.html

Reply:
I'm still with doubts about this suggestion since the main purpose for the `StatusDropdown` is not just to be a `Menu` with different `MenuItem` childrens, but it is more generic. It could be used to simply display a list of items, but only for informational purposes, or simply display an image or some behavior that is not specifically of a 'Menu'. It's a simple `Popup` container where you can customize its `contentItem`.